modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Review/index.tsx

```diff
@@ -130,7 +163,7 @@ const Review = () => {
 
 						<ClayTable.Cell className="col" headingCell>
 							<span className="text-dark">
-								<h6>$0.00</h6>
+								<h6>$611.00</h6>
```

Comment:
I believe that you should not mock this value. It should be a sum of the values of the column.